src/content/docs/accounts/accounts/automated-user-management/automated-user-provisioning-single-sign.mdx

```diff
@@ -32,8 +32,8 @@ Requirements and impacts:
 
 * Requires Enterprise edition.
 * [User model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model#user-models)-related requirements:
-  * This feature creates users on our [New Relic One user model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model#user-models).
-  * To implement AUM, you must be on our [New Relic One user model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model#user-models) and have [user management roles assigned](/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles#add-user-managers). If you're on our [original user model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model#user-models) (or otherwise can't seem to implement this feature), talk to your New Relic account representative.
+  * This feature requires you to be on our [New Relic One user model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model#user-models) and creates users on that model. If you're on our [original user model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model#user-models) (or otherwise can't seem to implement this feature), talk to your New Relic account representative.
+  * Implementing AUM requires the [**Authentication domain manager** and the **Organization manager** role](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure/#standard-roles) (both available with the [default group **Admin**](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure/#groups)).
```

Comment:
Implementing this requires a specific user/account has these roles, right? I think it would be helpful to clarify that.

Reply:
Also, this parenthetical feels a little odd to me: "(both available with the default group Admin)"

Would it be more helpful to say something like, "users in the default group Admin are assigned these roles"?